app/src/main/res/values/strings.xml

```diff
@@ -257,6 +252,10 @@
         opens the Firfox Reality support web site in the browser window. -->
     <string name="settings_help">Help</string>
 
+    <!-- This string is displayed in the settings header under the Firefox Reality logo. When clicked the settings panel is closed
+         a the survey web site in opened in the currently focused window -->
+    <string name="settings_send_your_feedback">&lt;a href="https://qsurvey.mozilla.com/s3/FxR">Send us Your Feedback&lt;/a></string>
+
```

Comment:
Is there a better way to handle the URL instead of including it in the l10n string? Seems like we should only have it in one place. Also, @jvonitter Do we have localized versions of this survey?

Reply:
Updated with separate link/text.